pkg/apis/core/types.go

```diff
@@ -572,13 +572,17 @@ type PersistentVolumeClaimStatus struct {
 	// +featureGate=RecoverVolumeExpansionFailure
 	// +optional
 	AllocatedResources ResourceList
```

Comment:
I just noticed that the container status has `ResourcesAllocated`: https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/core/types.go#L2500

We should consider aligning the field names.